tooling/api/src/fs.ts

```diff
@@ -167,12 +163,9 @@ function arrayBufferToBase64(buffer: ArrayBuffer): string {
 /**
  * Writes a binary file
  *
- * @param {FsBinaryFileOption} file
- * @param {string} file.path Path of the file
- * @param {ArrayBuffer} file.contents Contents of the file
- * @param {FsOptions} [options] Configuration object
- * @param {BaseDirectory} [options.dir] Base directory
- * @returns {Promise<void>}
+ * @param file File configuration object
+ * @param [options]
```

Comment:
What's the `[options]` for?

Reply:
in the `fs` module, there is a lot of [options] which are general configuraion object. I think I skipped those as typedoc doesn't seem to generate description for optional args.